.gitignore

```diff
@@ -444,3 +444,6 @@ $RECYCLE.BIN/
 *.lnk
 
 # End of https://www.toptal.com/developers/gitignore/api/linux,windows,intellij+all,microsoftoffice,latex
+
+# For repo 
+/*.csv
```

Comment:
Why this? I think, this has to be removed - we do version `csv` files.

Reply:
This ignore only affects csv files in the root directory of the repository, which are usually generated by script combining or processing, and in fact are not checked into the repository. journals/*.csv is not affected.